internal/capability/report.go

```diff
@@ -81,21 +81,21 @@ func (opt OperatorInstallRptOptionFile) report(ca CapAudit) error {
 // Simple print option implmentation for operand install
 type OperandInstallRptOptionPrint struct{}
 
-func (OperandInstallRptOptionPrint) report(ca CapAudit) error {
+func (OperandInstallRptOptionPrint) report(ca capAudit) error {
```

Comment:
Actual output should probably be limited to the cmd package. I would suggest changing the signature here to accept an io.Writer, and using Fprint. This will allow for future tests to be written.

Reply:
Totally agree with the io.Writer and Fprint that can allow for testing and things way more independent. I'm on it. Just one quesion: what do you mean with

> Actual output should probably be limited to the cmd package.? @bcrochet

Reply:
Hi @bcrochet , here is the change f1a3a82. I hope it's something in those lines you're thinking. We indeed need to work on better reporting. This is very temporary and needs a better system. But now they take io.writer which allows for testing.